### ISSUE TITLE
Certificate enrollment?

### ISSUE THREAD

**Issue by 2relativ** [aruba/aoscx-ansible-collection#103]
Hey guys, just noticed another thing which might be useful in the future for me. To use the really cool feature of downloading user roles (DUR) from clearpass it is common sense to install the trusted root certificate of your organization or one that clearpass also trusts to use https. 
https://www.flomain.de/2022/06/aruba-downloadable-user-roles/. And since there is also a clearpass ansible library it might be cool to have everything in one playbook to enable DUR for a switch.
It would also be cool to use https with ansible with a trusted cert. Not everybody might have a certificate enrollment program i.e. ACME certbots which not only requests certs but also installs them to necessary systems. So to get the communication easily secure, it would be cool if ansible could also install the certs to a switch. Just a proposal. Also would be cool (if the proposal persists) to make the necessary options to properly use DURs also configurable.

**Comment by tchiapuziowong** [comment]
Hi @2relativ thank you so much for the suggestion! Is this a playbook you've already created? If so you're more than welcome to open a Pull Request on our AOS-CX Ansible Workflows project where we host useful workflows for users utilizing the CX Collection: https://github.com/aruba/aoscx-ansible-workflows

**Comment by 2relativ** [comment]
No, I have not. Is there a way yet to import certs except for maybe using the command module?
But if I find a way, I will maybe. Love the possibilities with ansible. Even considering learning python.

**Comment by tchiapuziowong** [comment]
At this time we only support certificate configuration using the SSH modules `aoscx_command` and `aoscx_config` - we have planned certificate support through APIs but are unable to provide an ETA at this time

**Comment by Lazzorides** [comment]
Hi,
Did somebody find already a solution? I am also facing this issue and it would be very great to implement it.
